test/core/update_taxon_assignments_test.py

```diff
@@ -0,0 +1,90 @@
+import os
+import time
+import unittest
+from configparser import ConfigParser
+from uuid import uuid4
+
+from GenomeFileUtil.GenomeFileUtilImpl import GenomeFileUtil
+from GenomeFileUtil.GenomeFileUtilServer import MethodContext
+from installed_clients.WorkspaceClient import Workspace as workspaceService
+
+# NOTE: These tests must run against https://ci.kbase.us
+_WORKSPACE_ID = 33192
+_OBJECT_ID = 33
+
+
+class UpdateTaxonAssignmentsTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        token = os.environ.get('KB_AUTH_TOKEN', None)
+        # WARNING: don't call any logging methods on the context object,
+        # it'll result in a NoneType error
+        cls.ctx = MethodContext(None)
+        cls.ctx.update({'token': token,
+                        'provenance': [
+                            {'service': 'GenomeFileUtil',
+                             'method': 'please_never_use_it_in_production',
+                             'method_params': []
+                             }],
+                        'authenticated': 1})
+        config_file = os.environ['KB_DEPLOYMENT_CONFIG']
+        cls.cfg = {}
+        config = ConfigParser()
+        config.read(config_file)
+        for nameval in config.items('GenomeFileUtil'):
+            cls.cfg[nameval[0]] = nameval[1]
+        cls.wsURL = cls.cfg['workspace-url']
+        cls.wsClient = workspaceService(cls.wsURL, token=token)
+        cls.serviceImpl = GenomeFileUtil(cls.cfg)
+        suffix = int(time.time() * 1000)
+        cls.wsName = "test_GenomeFileUtil_" + str(suffix)
+        cls.wsClient.create_workspace({'workspace': cls.wsName})
+
+    def test_update_taxon_assignments_valid(self):
```

Comment:
This tests all happy paths for the methods. I couldn't think of any very useful error test cases.

Reply:
seems good to me, not sure what else we would test. 
One thing Im not sure about is if we should be testing the validity of any taxon assignments

Reply:
Good question, i'll ask Shanavir